proto/src/lib.rs

```diff
@@ -31,3 +39,11 @@ pub fn ntx_builder_api_descriptor() -> FileDescriptorSet {
     FileDescriptorSet::decode(&bytes[..])
         .expect("bytes should be a valid file descriptor created by build.rs")
 }
+
+/// Returns the Protobuf file descriptor for the worker status API.
+#[cfg(feature = "internal")]
+pub fn worker_status_api_descriptor() -> FileDescriptorSet {
```

Comment:
Not for this PR, but I think "worker status" is somewhat ambiguous. We should probably call it "prover worker" or something like that - but I think there is a different issue for this.

Reply:
Should this comment be addressed with #993 ?